packages/cashier/src/containers/routes/route-with-sub-routes.tsx

```diff
@@ -26,12 +35,11 @@ const RouteWithSubRoutes = route => {
         } else if (route.is_authenticated && !route.is_logging_in && !route.is_logged_in) {
             redirectToLogin(route.is_logged_in, getLanguage());
         } else {
-            const default_subroute = route.routes ? route.routes.find(r => r.default) : {};
-            const has_default_subroute = !isEmptyObject(default_subroute);
+            const default_subroute: TDefaultSubroute = route.routes?.find(r => r.default);
             const pathname = removeBranchName(location.pathname);
             result = (
                 <React.Fragment>
-                    {has_default_subroute && pathname === route.path && <Redirect to={default_subroute.path} />}
+                    {!!default_subroute && pathname === route.path && <Redirect to={default_subroute.path} />}
```

Comment:
@george-usynin-binary I don't this these 2 are equal in case the object is empty
`!!default_subroute !== !isEmptyObject(default_subroute)`